src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2190,4 +2202,15 @@ PeerImp::getScore (bool haveItem)
    return score;
 }
 
+bool
+PeerImp::highLatency ()
```

Comment:
@vinniefalco I could see your point if this inheritance were for truly intended for polymorphism -- adding the `const` could constrain potential implementations.  But I'm not seeing that level of polymorphism here (although I may have missed it).  To me it appears the inheritance is purely for information hiding: I only see one class derived from `Peer`, and it's `PeerImpl`.

Additionally, `Peer` already has a number of `const virtual` methods: `getRemoteAddress`, `id`, `cluster`, `getNodePublic`.  Are you suggesting those should not be `const`?  I'm not arguing.  I'm just trying to get a sense for the motivation behind your statement.